Symfony/CS/Events/FixerConfigurationResolvedEvent.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Events;
+
+use Symfony\Component\EventDispatcher\Event;
+use Symfony\CS\ConfigInterface;
+
+/**
+ * Event that is fired when the configuration of a Fixer has been resolved.
+ */
```

Comment:
I think we can do without it ;)
(https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md#82-author it is not required)